storagenode/pieces/readwrite_test.go

```diff
@@ -24,10 +29,10 @@ func BenchmarkReadWrite(b *testing.B) {
 
 	dir, err := filestore.NewDir(ctx.Dir("pieces"))
 	require.NoError(b, err)
-	blobs := filestore.New(dir)
+	blobs := filestore.New(dir, zap.NewNop())
```

Comment:
This order still feels weird to me, due to inconsistency with all the other packages.